http/src/main/resources/logback.xml

```diff
@@ -27,6 +27,10 @@
         </encoder>
     </appender>
 
+    <appender name="async-console" class="ch.qos.logback.classic.AsyncAppender">
```

Comment:
The cats-effect blocked-thread-detector was complaining about log4cats. I discovered we were using the synchronous appender, which does blocking file I/O. I changed it to an `AsyncAppender`, which writes log messages to a queue, which a background threads pulls and writes to disk.